cloud-consumption-api/src/main/java/com/sequenceiq/consumption/api/v1/consumption/endpoint/ConsumptionInternalEndpoint.java

```diff
@@ -41,4 +42,12 @@ void scheduleStorageConsumptionCollection(@AccountId @QueryParam("accountId") St
     void unscheduleStorageConsumptionCollection(@AccountId @QueryParam("accountId") String accountId,
             @NotNull @ValidCrn(resource = {CrnResourceDescriptor.ENVIRONMENT, CrnResourceDescriptor.DATALAKE})
             @QueryParam("monitoredResourceCrn") String monitoredResourceCrn, @NotEmpty @QueryParam("storageLocation") String storageLocation);
+
+    @GET
+    @Path("exists/storage")
+    @Produces(MediaType.APPLICATION_JSON)
+    @ApiOperation(value = ConsumptionOpDescription.UNSCHEDULE_STORAGE, produces = MediaType.APPLICATION_JSON, nickname = "unscheduleStorageCollection")
+    Boolean doesStorageConsumptionCollectionExist(@AccountId @QueryParam("accountId") String accountId,
```

Comment:
is `Boolean` enough future proof? shouldn't be there a result object with a boolean for now so it could be extended?

Reply:
Added result object